services/movie.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+from db.models import Genre, Movie, Actor
+
+
+def get_movies(
+        genres_ids: List[Genre] = None,
```

Comment:
add Optional here and other place where argument can be None

Reply:
"Second, it also Optional because default parameter is None"
I can't understand what it means